message_definitions/v1.0/messages.xml

```diff
@@ -2208,8 +2208,14 @@
       <field name="yaw_c"  type="float"/>
   </message>
 
-  <!-- 251 is free -->
-  <!-- 252 is free -->
+   <!--251 is free -->
+
+   <message name="GPS_SMALL" id="252">
+    <field name="multiplex_speed"  type="uint32">bits 31-21 course in decideg : bits 20-10 ground speed in cm/s : bits 9-0 climb speed in cm/s</field>
+    <field name="lat"  type="int32" unit="1e7deg" alt_unit="deg" alt_unit_coef="0.0000001"></field>
+    <field name="lon"  type="int32" unit="1e7deg" alt_unit="deg" alt_unit_coef="0.0000001"></field>
+    <field name="alt"    type="int16" unit="cm" alt_unit="m" alt_unit_coef="0.01">height above the ellipsoid</field>
```

Comment:
btw, `alt_unit_coef` is not needed for the ones listed in message_definitions/common/units.xml